src/solvers/dgsem/basis_lobatto_legendre.jl

```diff
@@ -553,7 +552,7 @@ function lagrange_interpolating_polynomials(x, nodes, wbary)
 end
 
 """
-    gauss_lobatto_nodes_weights(n_nodes::Integer)
+    gauss_lobatto_nodes_weights(RealT, n_nodes::Integer)
```

Comment:
```suggestion
    gauss_lobatto_nodes_weights([RealT=Float64], n_nodes::Integer)
```

Reply:
Just a minor remark, but I would include the comma in the brackets
```suggestion
    gauss_lobatto_nodes_weights([RealT=Float64,] n_nodes::Integer)
```
because we do that elsewhere, e.g., in the docstring of `convergence_test`.